### PR TITLE
added rabbitmq

### LINKED ISSUE #23: Add RabbitMQ
Add RabbitMQ as an optional software to ansible

### PR DESCRIPTION
**Pull request by plandolt**
resolves https://github.com/vagrant-php/ubuntu/issues/23

### CODE REVIEW

File: ansible/roles/rabbitmq/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+- name: install plugins
+  rabbitmq_plugin: names={{ item }} state=enabled
+  with_items:
+    - rabbitmq_management
+  notify:
+  - restart rabbitmq
+
+- name: add user admin
+  rabbitmq_user: user=admin password=admin configure_priv=.* read_priv=.* write_priv=.* tags=administrator state=present
```
Comment (dominikzogg) [inline ansible/roles/rabbitmq/tasks/main.yml]: @scuben please use dynamic values for user/password (defined in vagrant-default.yml)

File: vagrant-default.yml [hunk 2]
```diff
@@ -64,6 +64,7 @@ role:
     postgresql: false
     redis: false
     blackfire: false
+    rabbitmq: false
```
Comment (dominikzogg) [inline vagrant-default.yml:67]: @scuben please provide add a rabbitmq with user / password and the config as yaml which can be coverted by a jinja template https://www.rabbitmq.com/configure.html#config-file-formats

**Review by dominikzogg — CHANGES_REQUESTED** [review]

**Comment by plandolt** [comment]
Added user/pass and config options. config is not yaml but sysctl format.

**Comment by dominikzogg** [comment]
@scuben you've done it as i meant it, thanks

**Comment by dominikzogg** [comment]
@scuben thanks for the contribution

**Pull request merged**